blockchain/contracts/HelloWorld.sol

```diff
@@ -25,15 +27,18 @@ contract HelloWorld {
         NFTOwnership memory nftOwnership;
         nftOwnership.nftId = nftHash;
         nftOwnership.owner = nftOwner;
+        nftOwnership.droptime = 1637085033;
```

Comment:
This should not be hardcoded but rather be a parameter of the mockNFT function

Reply:
Good idea!